pkg/kubelet/container/runtime.go

```diff
@@ -364,14 +364,23 @@ type PortMapping struct {
 	HostIP string
 }
 
+type DeviceInfo struct {
+	// Path on host for mapping
+	PathOnHost string
+	// Path in Container to map
+	PathInContainer string
+	// Cgroup permissions
+	Permissions string
```

Comment:
Apologies. I confused this with CRI. BTW I don't think we can make arbitrary changes to the Runtime interface since Kubelet is introducing a remote client API for runtimes. This change might have to be reflected there as well.  Look [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/api/v1alpha1/runtime/api.proto) for the new API.